styles/stylus/catppuccin.user.css

```diff
@@ -169,41 +169,46 @@
     }
     .cm-s-default {
       .cm-string {
-        color: @peach;
+        color: @green;
       }
       .cm-qualifier {
-        color: @peach;
+        color: @yellow;
       }
       .cm-variable-2 {
-        color: @sapphire;
+        color: @mauve;
       }
       .cm-keyword {
         color: @mauve;
       }
-      .cm-atom,
+      .cm-atom {
+        color: @red;
+      }
       .cm-number {
-        color: @green;
+        color: @peach;
       }
       .cm-def {
-        color: @sapphire;
+        color: @mauve;
       }
       .cm-tag {
-        color: @sapphire;
+        color: @blue;
       }
       .cm-builtin {
-        color: @subtext1;
+        color: @red;
       }
       .cm-variable-3 {
-        color: @subtext1;
+        color: @teal;
       }
       .cm-comment {
-        color: @subtext0;
+        color: @overlay2;
       }
       .cm-operator {
         color: @yellow;
       }
       .cm-variable {
-        color: @text;
+        color: @blue;
```

Comment:
Feels like normal variables should be kept as `@text` unless I'm misunderstanding what `.cm-variable` actually highlights.

Reply:
Codemirror isn't the most accurate but through testing this does result in the most accurate syntax highlighting in Stylus. Feel free to try it out by editing one of these userstyles, since all Stylus is used for really is CSS. Happy to update if you spot anything wrong visually.

Reply:
Ah of course it's CSS variables, which are blue elsewhere.
My bad :+1: